config/config.go

```diff
@@ -508,6 +511,14 @@ func (cfg Config) Validate() error {
 		)
 	}
 
+	if cfg.Twilio.VoiceName != "" {
+		err = validate.Many(
+			err,
+			validate.ASCII("Twilio.VoiceName", cfg.Twilio.VoiceName, 1, 50),
+			validate.ASCII("Twilio.VoiceLanguage", cfg.Twilio.VoiceLanguage, 1, 10), // language is required when a voice name is set
+		)
+	}
```

Comment:
Don't let language escape validation if name is unset

```suggestion
	if cfg.Twilio.VoiceName != "" {
		err = validate.Many(
			err,
			validate.ASCII("Twilio.VoiceName", cfg.Twilio.VoiceName, 1, 50),

			// language is required when a voice name is set
			validate.ASCII("Twilio.VoiceLanguage", cfg.Twilio.VoiceLanguage, 1, 10),
		)
	} else {
		err = validate.Many(
			err,
			validate.ASCII("Twilio.VoiceLanguage", cfg.Twilio.VoiceLanguage, 0, 10),
		)
	}
```

Reply:
applied.